product/README.md

```diff
@@ -0,0 +1,74 @@
+# Product @ DVELP
+Our Playbook for creating products at DVELP.
+
+## Our Mission
+To create solutions that help businesses revolutionise how they speak with
+their customers.
+
+## Strategy
+To [identify White Space](#fill-the-white-space) in [Partner Solutions](/partner-solutions),
+[Prototype rapidly](#get-close-with-customers) and validate hypothesis amongst
+our customer base. Pick winners early and [scale up](#piggyback-on-partners)
+through strong vendor partner relationships.
+
+### Get Close with Customers
+Through close relationships with customers we can deliver new features and
+prototypes to rapidly validate hypotheses.
+
+### Fill the White Space
+Leverage insight from customers to recognise industry trends early and identify
+opportunities to fill `White Space` left by
+[Partner Solutions](partner-solutions).
+
+### Piggyback on Partners
+Build a strong commercial and go-to-market framework with our
+[Partner Vendors](partner-solutions) through which we can scale-up products
+that have shown early success with existing customers.
+
+## Product Team
+* Chief Product Officer: Tom Mullen
+* Senior Vice President of Product: Damian Kellly
+* Vice President of Product Engineering: TBC
+
+## What We Believe In
+We believe, above all else, in the following principles, which act as guides to
+help us build incredible products:
+
+**Focus Above the Value Line**
+We spend as much time and focus as we can on things that will add value to our
+customers and to the end user.
+
+**Ship Early and Often**
+We don't like big bang releases. We break down complex tasks and ship small
+features regularly to get early feedback and reduce risk.
+
+**We Are About Quality**
+At our core, we are about quality. We obsess over detail.
+
+
+## Resources
+* [Product Strategy Sheet](https://docs.google.com/spreadsheets/d/16UzzS_aGoPzv1ThoknyqM4NJ3k09QN5M3B7DtxIKY2o)
+* [Product Estate](https://docs.google.com/spreadsheets/d/1YR6lM9HXHw2rkY1h2xiqOByDMlc0-1attTKJflA7A9w)
+
+### Org Structure
+![alt text](https://raw.githubusercontent.com/DVELP/cookbook/product/assets/product-org-structure.png "DVELP Product Org")
+
+## Scope of Ownership
+The product team is responsible for both home-grown `Product` and `Partner
+Solutions`. More information about each can be found below.
+
+* [Product](product)
```

Comment:
`cookbook/product/product` is a confusing namespace

Reply:
Changed